pkg/cmd/triggertemplate/list.go

```diff
@@ -87,7 +86,11 @@ or
 			}
 
 			if output != "" {
-				return printer.PrintObject(stream.Out, tts, f)
+				p, err := f.ToPrinter()
```

Comment:
can we keep this also as function like before, it is getting duplicated at multiple places

Reply:
this comment is invalid based on above comment reply